rollup-common.js

```diff
@@ -63,7 +64,11 @@ const crossPackagePropertyMangles = {
   _element: "E",
   _options: "O",
   _template: "t",
-  _renderOptions: "R"
+  _renderOptions: "R",
+  _baseConnectedCallback: "bC",
+  _baseUpdate: "bU",
```

Comment:
Nit: the naming here (also, `_template: t`) doesn't match the description above:
> Mangled names are ***randomly chosen uppercase letters***, in case we ever might want to use lowercase letters for short, public APIs.